client/src/components/Applications/AppForm/AppForm.js

```diff
@@ -207,9 +203,9 @@ export const AppSchemaForm = ({ app }) => {
                   &nbsp;to request access.
                 </>
               )}
-            </Alert>
+            </Message>
```

Comment:
See earlier comment for opening tag.

```suggestion
            </SectionMessage>
```